### DISCUSSION TITLE
How to use Anthropic's Claude as the LLM model?

### DISCUSSION THREAD

**Discussion by mflisiuk** [ALwrity/ALwrity#141] [General]
Hi there!

First of all, I really like your tool and its capabilities. However, I'm interested in using Anthropic's Claude as the LLM model instead of the default OpenAI models. 

I've looked through the documentation but couldn't find any information about:
- Whether Claude integration is supported
- How to configure the tool to use Claude API
- What specific settings or environment variables are needed

Could you please provide some guidance on:
1. Is Claude integration currently supported?
2. If yes, what are the steps to set it up?
3. If not, are there any plans to add Claude support in the future?

I'm particularly interested in Claude because of its [you can mention specific features you're interested in, e.g., "longer context window", "specific capabilities", etc.]

Thanks in advance for any information!

**Comment by AJaySi** [comment]
Hi,

1). Chatgpt and Gemini give good content generation.
2). Presently, ALwrity prompts are tested for chatgpt & Gemini.

3). Integration claude APIs are easy enough, but testing and tuning the prompts to get good content takes time.

Yes, we have plans to integrate Claude in the future.

Note: Presently, we are busy building a SaaS webapp for Alwrity with Next.js and this python streamlit version is suffering.

Thank  you for your kind words.
